src/redempt/redlib/region/CuboidRegion.java

```diff
@@ -317,6 +318,15 @@ public CuboidRegion getFace(BlockFace face) {
 		region.expand(face.getOppositeFace(), -region.measure(face) + 1);
 		return region;
 	}
+
+	public Block getRandomBlock() {
+		Random random = new Random();
+		int[] limits = getBlockDimensions();
+		int x = start.getBlockX()+random.nextInt(0, limits[0]);
+		int y = start.getBlockY()+random.nextInt(0, limits[1]);;
```

Comment:
Extra semicolon on this line and the following one

Reply:
Fixed in the newer commit